test/test_create_model.py

```diff
@@ -0,0 +1,32 @@
+from training.creating_dataset import create_dataframe
+from training.create_model import create_model
+from training.splitting_dataset import get_random_split_results
+from sklearn.linear_model import RidgeCV
+from sklearn.ensemble import RandomForestRegressor
+
+
+def test_linear_regression():
+    dataset = create_dataframe()
+    X_train, X_test, y_train, y_test = get_random_split_results(dataset)
+    X_train = X_train.drop(columns=["book"])
```

Comment:
Proponuję użyć tutaj jakichś mockowych danych, zamiast ładowania całości. Można to zamknąć jako jeden fixture PyTesta. Scikit-learn też tak robi swoje testy, że budują małe sztuczne dane na kilka wierszy